src/main/java/org/sagebionetworks/web/client/widget/team/JoinTeamWidget.java

```diff
@@ -79,6 +82,8 @@ public JoinTeamWidget(JoinTeamWidgetView view,
 		this.jsonObjectAdapter = jsonObjectAdapter;
 		this.gwt = gwt;
 		this.wikiPage = wikiPage;
+		this.progressWidget = progressWidget;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/8346437/9453588/467687ce-4a70-11e5-8338-e2bffa00c7a3.png)

Needed to extend UiObject, which is where addStyleNames comes from.